rel/expr_tuple.go

```diff
@@ -142,12 +142,18 @@ func (e *TupleExpr) String() string { //nolint:dupl
 func (e *TupleExpr) Eval(ctx context.Context, local Scope) (Value, error) {
 	tuple := EmptyTuple
 	var err error
+	errs := Errors{}
```

Comment:
`var err` (line above) should be declared where it's used, not at the top of the function.

Reply:
The following loop will use it, so declared here.

Reply:
It is used in following loop with code:
```
	for _, attr := range e.attrs {
		tuple, err = attr.Apply(ctx, local, tuple)
```
I think current code is better, as `tuple` has to be declared in the top of func. Or it has to add code to transfer  looks like:
```
	for _, attr := range e.attrs {
		interTuple, err := attr.Apply(ctx, local, tuple)
                tuple = interTuple
```